adam/language_specific/english/english_language_generator.py

```diff
@@ -116,7 +137,7 @@ def generate(self) -> ImmutableSet[LinearizedDependencyTree]:
             )
 
         def _translate_object_to_noun(
-            self, _object: SituationObject
+            self, _object: SituationObject, count: int = 1
```

Comment:
It is not necessary on this pull request, but make an issue (and reference it from a comment in the code) to eventually enforce that when using plurals we need to ensure that all relevant relations and actions involving any of the pluralized things involve them all (e.g. we can't say _The two boxes on the table_ if one box is under the table)

Reply:
Described issue in a comment

Reply:
Thanks - can you make a match issue on GitHub and include its URL in the comment also?